packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -37,16 +34,14 @@ const HelperMessage = ({ message, isError }: HelperMessageProps) => (
 )
 
 type BalanceTextFieldProps = {
-  balance: number
+  balance: number | undefined
   maxBalance?: number
-  label?: string
   isError: boolean
-  onChange: (balance: number) => void
+  onBlur: (balance: number | undefined) => void
```

Comment:
is onBlur the right name here? Maybe onChange is more correct, or onCommit as you used in the other component

Reply:
I actually considered renaming the `onCommit` to `onBlur` as well 🙈 
But I'll change the blur to commit for consistency